src/Dotnet.Script.Core/Dotnet.Script.Core.csproj

```diff
@@ -31,7 +31,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="2.4.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Scripting" Version="2.6.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.0.0" />
```

Comment:
Should we bump this one too?
It's available as 2.0.3 from NuGet now
https://www.nuget.org/packages/Microsoft.Extensions.DependencyModel/

Reply:
Good point